download_and_create_reference_datasets/v02/mito/write_mito_mitomap_ht.py

```diff
@@ -0,0 +1,19 @@
+import hail as hl
+
+from download_and_create_reference_datasets.v02.mito.utils import load
+
+CONFIG = {
+    # The data source is a web page. So we download it manually and save them to the GCS.
```

Comment:
this comment is insuffiecient. It needs to be detailed and include the exact URL where the data was copied from